frontend/src/scenes/insights/views/WorldMap/countryCodes.ts

```diff
@@ -289,3 +301,189 @@ export const countryCodeToName = {
     UM: 'United States Minor Outlying Islands',
     YT: 'Mayotte',
 }
+
+export const languageCodeToName: Record<string, string> = {
```

Comment:
Maybe add a comment about where these came from. You might want separate sections for the standard 2 letter ISO country codes, and then a separate sections for dealing with unexpected values seen in the wild